app/models/user.rb

```diff
@@ -3,6 +3,10 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  enum gender: {male: 0, female: 1, other: 2}
+
+  scope :recent, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent, -> {order(created_at: :desc)}
```
Thêm space trước block e nhé

Reply:
Huhu anh ơi, cái này rubocop nó tự check identation xong nó xóa ạ :<<

Reply:
Chỗ này e sửa rule của rubocop phần space trước block này được nha